### PR TITLE
gnome-shell: Sync with 3.34.2 changes

### PR DESCRIPTION
**Pull request by 3v1n0**
Let's keep tracking stable version for now

### CODE REVIEW

**Comment by clobrano** [comment]
For reference, the main changes I noted are

- today in calendar is a full circle instead than just a border
- selected color border around circular buttons (close, shutdown, etc...), which looks a bit odd

but it looks good to me overall

**Comment by Feichtmeier** [comment]
I think we might need the corresponding shell version to test this.

Do you have a deadline for this to be merged @3v1n0?

**Comment by clobrano** [comment]
This is for a SRU, so I think it is expected to work with 3.32 as well

https://bugs.launchpad.net/ubuntu/+source/gnome-shell/+bug/1857037

File: gnome-shell/src/gnome-shell-sass/_common.scss [hunk 1]
```diff
@@ -1216,12 +1230,11 @@ StScrollBar {
     &:hover, &:focus {
       background-color: $_hover_bg_color;
       color: $fg_color;
-      border: none;
-      padding: 14px;
     }
     &:active { 
       background-color: $_active_bg_color;
       color: $fg_color;
+      border-color: $selected_borders_color;
```
Comment (Feichtmeier) [inline gnome-shell/src/gnome-shell-sass/_common.scss]: Ah, no this makes sense for upstream but not for us. @3v1n0 Could you remove this line again?
They have blue system menu disks so this makes sense for them

**Comment by iainlane** [comment]
I have just pushed 3.34.3 to focal, so it would be good to get this PR tidied up soon.

And I'd say it's required before uploading the SRU to eoan.

File: gnome-shell/src/gnome-shell-sass/_common.scss (hunk 1 above)
Comment (3v1n0) [reply gnome-shell/src/gnome-shell-sass/_common.scss]: Ok, sorry for being late, but you know... Holidays :)

Anyways, fixed now.

**Comment by 3v1n0** [comment]
> And I'd say it's required before uploading the SRU to eoan.

@iainlane agree, the eoan (and actually ubuntu/master too) for eoan is then at 
https://github.com/ubuntu/yaru/tree/ubuntu/eoan

**Comment by Feichtmeier** [comment]
Working - thanks!
grafik

**Pull request merged**